great_expectations/data_context/types/base.py

```diff
@@ -1319,6 +1320,66 @@ def config_version(self):
         return self._config_version
 
 
+class LegacyCheckpointConfig(BaseYamlConfig):
+    def __init__(
+        self,
+        module_name: Optional[str] = None,
+        class_name: Optional[str] = None,
+        validation_operator_name: Optional[str] = None,
+        batches: Optional[list] = None,
+        commented_map: Optional[CommentedMap] = None,
+    ):
+        super().__init__(commented_map=commented_map)
+        self._module_name = module_name or "great_expectations.checkpoint"
+        self._class_name = class_name or "LegacyCheckpointConfig"
+        self._validation_operator_name = (
+            validation_operator_name or "action_list_operator"
+        )
+        self._batches = batches or []
+
+    @property
+    def module_name(self):
+        return self._module_name
+
+    @property
+    def class_name(self):
+        return self._class_name
+
+    @property
+    def validation_operator_name(self):
+        return self._validation_operator_name
+
+    @property
+    def batches(self):
+        return self._batches
+
+    @classmethod
+    def get_config_class(cls):
+        raise cls  # LegacyCheckpointConfig
+
+    @classmethod
+    def get_schema_class(cls):
+        raise LegacyCheckpointConfigSchema
+
+
+class LegacyCheckpointConfigSchema(Schema):
+    class Meta:
+        unknown = INCLUDE
+
+    class_name = fields.Str()
+    module_name = fields.Str()
+    validation_operator_name = fields.Str()
+    batches = fields.List(
+        fields.Dict(
+            keys=fields.Str(validate=OneOf(["batch_kwargs", "expectation_suite_names"]))
+        )
+    )
+
+    @post_load
+    def make_legacy_checkpoint_config(self, data, **kwargs):
```

Comment:
@roblim Let's delete the `@post_load` for `LegacyCheckpointConfigSchema` also.  Thanks.